### PR TITLE
Fix: Windows supports only two options on dialog boxes

### PR DESCRIPTION
**Pull request by seromenho**
For https://github.com/Fliplet/fliplet-studio/issues/2229

### CODE REVIEW

File: js/components/image.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ Fliplet.FormBuilder.field('image', {
       return new Promise(function(resolve, reject) {
         $vm.boundingRect = fileInput.getBoundingClientRect();
 
+        var buttonLabels = ["Take Photo", "Choose Existing Photo", "Cancel"];
+		    if (Modernizr.windows) {
```
Comment (hcarneiro) [inline js/components/image.js]: Wrong indentation

Comment (seromenho) [reply js/components/image.js]: wtf? This is what I have here: https://cl.ly/1u1I0d1M2l0V

Comment (seromenho) [reply js/components/image.js]: fixed.

File: js/components/image.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ Fliplet.FormBuilder.field('image', {
         $vm.boundingRect = fileInput.getBoundingClientRect();
 
         var buttonLabels = ["Take Photo", "Choose Existing Photo", "Cancel"];
-		    if (Modernizr.windows) {
+        if (Modernizr.windows) {
           var buttonLabels = ["Take Photo", "Choose Existing Photo"];
```
Comment (hcarneiro) [inline js/components/image.js]: Why are you declaring `var` again?

**Pull request merged**